Windows install: u_int not declared

On Windows with GNU Octave 10.1.0, 

```octave
pkg install -forge datatypes
```

results in the following errors. 
Might `unsigned int` be suitable instead of `u_int` in general?

On Linux, type "u_int" is declared in <sys/types.h>, but not on Windows.

> __datetime__.cc: In function 'RowVector seconds2vector(double, std::string)':
__datetime__.cc:89:13: error: 'u_int' was not declared in this scope
   89 |   OUT(1) = (u_int)date_tp.month();
      |             ^~~~~
__datetime__.cc: In function 'RowVector tz2vector(const ZonedType&, std::string)':
__datetime__.cc:110:13: error: 'u_int' was not declared in this scope
  110 |   OUT(1) = (u_int)date_local.month();
      |             ^~~~~
__datetime__.cc: In function 'RowVector timezone2vector(const ZonedType&)':
__datetime__.cc:157:13: error: 'u_int' was not declared in this scope
  157 |   OUT(1) = (u_int)date_tp.month();
      |             ^~~~~
__datetime__.cc: In function 'octave_value_list F__datetime__(const octave_value_list&, int)':
__datetime__.cc:810:19: error: 'u_int' was not declared in this scope
  810 |           M(i) = (u_int)ymd.month();
      |                   ^~~~~
__datetime__.cc:892:19: error: 'u_int' was not declared in this scope
  892 |           M(i) = (u_int)ymd.month();

--- a/src/Makefile
+++ b/src/Makefile
@@ -1,7 +1,7 @@
 # Makefile for compiling required oct files
 
 ifeq ($(OS),Windows_NT)
-	LIBS = -lOle32
+	LIBS = -lOle32 -lcurl
 else
 	LIBS =
 endif

--- a/src/date/tz.h
+++ b/src/date/tz.h
@@ -49,7 +49,7 @@
 
 #ifndef HAS_REMOTE_API
 #  if USE_OS_TZDB == 0
-#    if defined _WIN32 || defined __ANDROID__
+#    if defined __ANDROID__
 #      define HAS_REMOTE_API 0
 #    else
 #      define HAS_REMOTE_API 1

